sql/sqlline-client/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.stargate.api.sql</groupId>
+    <artifactId>sql-base</artifactId>
+    <version>0.0.16-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.api.sql.client</groupId>
+  <artifactId>stargate-sqlline-client</artifactId>
```

Comment:
Similar to above, thoughts about `sql-client`?

Reply:
ok.

Reply:
renamed to `sql-shell` because it is really a shell :)